### DISCUSSION TITLE
Embedding queries? Or: how to see tasks from the repository ordered by date?

### DISCUSSION THREAD

**Discussion by JenFau** [foambubble/foam#1331] [Ideas]
Is there a way to embed a query onto a note or template? I am using Foam to take notes and actions in meetings (one note per meeting). I want the action to remain in the note for the meeting, but to surface them in various ways, ideally in a 'today', 'tomorrow', 'next week' kind of structure. 

ToDo tree works great for pulling the tasks out, but unless I put the due date at the front of the task, it doesn't do date sorting. 

Is there a way of embedding queries into a template so that they are automatically populated when a note is created?

**Comment by riccardoferretti** [comment]
Foam doesn't support tasks as a first class citizen, so there are no features specifically geared at what you are asking.
So for that you'd need to check what the extension specifically (in your case ToDo tree, but there are others) supports

**Comment by stereobooster** [comment]
related https://github.com/foambubble/foam/discussions/1312
